stuff/pipelines/contest_callback.py

```diff
@@ -7,15 +7,20 @@
 import logging
 
 from typing import List, Optional
-
-import verification_code
-
-import omegaup.api
-import pika
 import mysql.connector
 import mysql.connector.cursor
 from mysql.connector import errors
 from mysql.connector import errorcode
+import pika
+
+import verification_code
+
+
+@dataclasses.dataclass
+class Ranking:
```

Comment:
Esto se podría sustituir por una tupla nombrada:
```suggestion
class Ranking(NamedTuple):
```
No olvides hacer el import desde `typing`